packages/engine/src/initialize.ts

```diff
@@ -119,6 +121,14 @@ export const initializeEngine = async (options): Promise<void> => {
   }
 
   if (!useOffscreen || options.editor) {
+    await AnimationManager.instance.getDefaultModel()
+    registerSystem(RaycastSystem);
+    registerSystem(StateSystem);
+    registerSystem(CharacterControllerSystem);
+    registerSystem(PhysicsSystem);
```

Comment:
seems to be a merge error here doubly registering systems.

Reply:
is this means to remove line 
`    registerSystem(PhysicsSystem);`
 `    registerSystem(TransformSystem, { priority: 900 });` 
 ?

Reply:
i've solved this in the latest commits